packages/components/src/components/hds/copy/button/index.ts

```diff
@@ -70,23 +85,25 @@ export default class HdsCopyButtonComponent extends Component {
   }
 
   @action
-  onSuccess(args) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  onSuccess(args: any) {
```

Comment:
@alex-ju would this be correct?
```suggestion
  onSuccess(args: ModifierSignature['Args']['Named']['onSuccess']) {
```

Reply:
yes, that would work too